packages/wallets/src/features/accounts/modules/Poa/components/POAMobile/POAMobile.tsx

```diff
@@ -0,0 +1,118 @@
+                )}
+                title={localize('Add a real MT5 account')}
+            >
+                <div className='wallets-poa-mobile-layout'>
+                    <div className='wallets-poa-mobile-layout__header'>
+                        <div className='wallets-timeline'>
+                            <LabelPairedArrowLeftMdBoldIcon onClick={() => setShouldRenderDocumentUpload(false)} />
+                            <Text align='start' as='p' lineHeight='sm' size='sm'>
+                                <Localize
+                                    components={[<strong key={0} />]}
+                                    i18n_default_text='<0>Step 2/2:&nbsp;</0> Upload proof of address'
+                                />
+                            </Text>
+                        </div>
+                        <div className='wallets-timeline__item'>
+                            <ProgressBar isActive />
+                            <ProgressBar isActive />
+                        </div>
+                    </div>
+                    <div className='wallets-poa-mobile-layout__container'>
+                        {errorSettings?.message && (
+                            <InlineMessage className='wallets-poa-mobile-layout__error-banner' variant='error'>
+                                <Text align='start'>{localize(errorSettings.message)}</Text>
+                            </InlineMessage>
+                        )}
+                        <DocumentSubmission countryCode={countryCode as string} />
+                    </div>
+                </div>
+            </ModalStepWrapper>
+        );
+    }
+
+    return (
+        <ModalStepWrapper
+            renderFooter={() => (
+                <Footer disableNext={isNextBtnDisabled} onClickNext={() => setShouldRenderDocumentUpload(true)} />
+            )}
+            title={localize('Add a real MT5 account')}
+        >
+            <div className='wallets-poa-mobile-layout'>
+                <div className='wallets-poa-mobile-layout__header'>
+                    <Text as='p' className='wallets-timeline' lineHeight='sm' size='sm'>
+                        <Localize
+                            components={[<strong key={0} />]}
+                            i18n_default_text='<0>Step 1/2:&nbsp;</0> Enter your address'
+                        />
+                    </Text>
+                    <div className='wallets-timeline__item'>
+                        <ProgressBar isActive />
+                        <ProgressBar />
+                    </div>
+                </div>
+                <div className='wallets-poa-mobile-layout__container'>
+                    {errorSettings?.message && (
+                        <InlineMessage className='wallets-poa-mobile-layout__error-banner' variant='error'>
+                            <Text>{localize(errorSettings.message)}</Text>
```

Comment:
I don't think this will be translated 🤔

Reply:
It was an existing wallets code. Let em ask the team